src/Stats/utils.js

```diff
@@ -0,0 +1,7 @@
+const sortDataByDate = (dataList) => {
+  const originalDataList = dataList.slice(0);
+  const orderedDataListByDateFromNearToFar = originalDataList.sort((a, b) => (b.date - a.date));
+  return orderedDataListByDateFromNearToFar;
+};
+
+export default sortDataByDate;
```

Comment:
lets export this as non default:
```js
export default = {
  sortDataByDate,
};
```

so it will be easier to export more methods later